app/src/main/java/com/khanhhuy/movie_db/screen/main/MainViewModel.java

```diff
@@ -0,0 +1,77 @@
+package com.khanhhuy.movie_db.screen.main;
+
+import android.app.Activity;
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.databinding.BindingAdapter;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.view.ViewPager;
+
+import com.khanhhuy.movie_db.adapter.ViewPagerAdapter;
+
+/**
+ * Created by yeu_thuong on 10/31/2017.
+ */
+
+public class MainViewModel extends BaseObservable implements MainContract.ViewModel {
+
+    private MainContract.Presenter mPresenter;
+    private MainActivity mMainActivity;
+
+    public MainViewModel(MainActivity mainActivity, MainContract.Presenter presenter){
+        mMainActivity = mainActivity;
+        mPresenter = presenter;
+    }
+
+    @Override
+    public void onStart() {
+        mPresenter.onStart();
+    }
+
+    @Override
+    public void onStop() {
+        mPresenter.onStop();
+    }
+
+    @Override
+    public void setPresenter(MainContract.Presenter presenter) {
+        mPresenter = presenter;
+    }
+
+    @BindingAdapter("bindViewPager")
+    public static void bindTabLayout(final TabLayout tabLayout, final ViewPager viewpager){
```

Comment:
có thể sử dụng hàm tablayout.setupWithViewPager

Reply:
https://android.jlelse.eu/tablayout-and-viewpager-in-your-android-app-738b8840c38a

Reply:
e nên định nghĩa 1 class BindingUtils sử dụng để @BIndingAdapter